### PR TITLE
Adds polymelia syndrome

### PR DESCRIPTION
**Pull request by MadmanMartian**
Polymelia syndrome - You grow more arms

### CODE REVIEW

**Comment by FudgePucker** [comment]
How do you wear a hardsuit while having an extra arm jutting out of your body?

**Comment by Aurx** [comment]
They weren't kidding when they said one size fits all.

File: code/modules/virus2/effect/effect.dm [hunk 1]
```diff
@@ -979,6 +979,31 @@ datum/disease2/effect/lubefoot/deactivate(var/mob/living/carbon/mob)
...
+	max_multiplier = 3
+	var/activated = FALSE
+
+/datum/disease2/effect/multiarm/activate(var/mob/living/carbon/mob)
+	if(activated)
+		return
+	var/hand_amount = round(multiplier)
+	mob.visible_message("<span class='warning'>With a spray of blood, [hand_amount > 1 ? "[hand_amount] more arms sprout" : "a new arm sprouts"]   from \the [mob]!</span>","<span class='notice'>[hand_amount] more arms burst forth from your back!</span>")
+	mob.set_hand_amount(mob.held_items.len + hand_amount)
+	blood_splatter(mob.loc,mob,TRUE)
```
Comment (SonixApache) [inline code/modules/virus2/effect/effect.dm]: phantom blood
`mob.vessel.remove_reagent("blood",4)` ?

Comment (MadmanMartian) [reply code/modules/virus2/effect/effect.dm]: Does blood splatter not do that already?

File: code/modules/virus2/effect/effect.dm [hunk 2]
```diff
@@ -979,6 +979,31 @@ datum/disease2/effect/lubefoot/deactivate(var/mob/living/carbon/mob)
...
+	name = "Polymelia Syndrome"
+	stage = 3
+	max_multiplier = 3
+	var/activated = FALSE
+
+/datum/disease2/effect/multiarm/activate(var/mob/living/carbon/mob)
+	if(activated)
+		return
+	var/hand_amount = round(multiplier)
+	mob.visible_message("<span class='warning'>With a spray of blood, [hand_amount > 1 ? "[hand_amount] more arms sprout" : "a new arm sprouts"]   from \the [mob]!</span>","<span class='notice'>[hand_amount] more arms burst forth from your back!</span>")
```
Comment (PJB3005) [inline code/modules/virus2/effect/effect.dm]: You got a duplicate space in here.

File: code/modules/virus2/effect/effect.dm (hunk 1 above)
Comment (PJB3005) [reply code/modules/virus2/effect/effect.dm]: If it doesn't, interacting with the mob vessel directly is a big no.

File: code/modules/virus2/effect/effect.dm [hunk 3]
```diff
@@ -979,6 +979,31 @@ datum/disease2/effect/lubefoot/deactivate(var/mob/living/carbon/mob)
...
+	blood_splatter(mob.loc,mob,TRUE)
+	activated = TRUE
+
+/datum/disease2/effect/multiarm/deactivate(var/mob/living/carbon/mob)
+	if(!activated)
+		return
+	var/hand_amount = round(multiplier)
+	mob.visible_message("<span class='notice'>The arms sticking out of \the [mob]'s back shrivel up and fall off!</span>", "<span class='warning'>Your new arms begin to die off, as the virus can no longer support them.</span>")
+	mob.set_hand_amount(mob.held_items.len - hand_amount)
+	for(var/i = 0; i < hand_amount;i++)
```
Comment (PJB3005) [inline code/modules/virus2/effect/effect.dm]: `for (var/i = 1 to hand_amount)`

**Review by PJB3005 — CHANGES_REQUESTED** [review]

File: code/modules/virus2/effect/effect.dm (hunk 1 above)
Comment (SonixApache) [reply code/modules/virus2/effect/effect.dm]: Oh. Do we have a proc for bloodloss? Because I messed with the vessel for drag damage.

**Review by PJB3005 — APPROVED** [review]

**Pull request merged**